hdp22/src/org/pentaho/hadoop/shim/hdp22/delegating/DelegatingHBaseConnection.java

```diff
@@ -0,0 +1,532 @@
+/*******************************************************************************
+*
+* Pentaho Big Data
+*
+* Copyright (C) 2002-2014 by Pentaho : http://www.pentaho.com
+*
+*******************************************************************************
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License. You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+******************************************************************************/
+
+package org.pentaho.hadoop.shim.hdp22.delegating;
+
+import org.pentaho.di.core.variables.VariableSpace;
+import org.pentaho.hbase.shim.api.ColumnFilter;
+import org.pentaho.hbase.shim.api.HBaseValueMeta;
+import org.pentaho.hbase.shim.hdp22.wrapper.HBaseConnectionInterface;
+import org.pentaho.hbase.shim.spi.HBaseBytesUtilShim;
+import org.pentaho.hbase.shim.spi.HBaseConnection;
+
+import java.util.List;
+import java.util.NavigableMap;
+import java.util.Properties;
+
+public class DelegatingHBaseConnection extends HBaseConnection implements HBaseConnectionInterface {
+  private final HBaseConnectionInterface delegate;
+
+  public DelegatingHBaseConnection( HBaseConnectionInterface delegate ) {
+    this.delegate = delegate;
+  }
+
+  /*
+   * (non-Javadoc)
+   *
+   * @see
+   * org.pentaho.hadoop.shim.mapr31.delegatingShims.HBaseConnectionInterface#addColumnFilterToScan(org.pentaho.hbase
+   * .shim.api.ColumnFilter, org.pentaho.hbase.shim.api.HBaseValueMeta, org.pentaho.di.core.variables.VariableSpace,
+   * boolean)
+   */
+  @Override
+  public void addColumnFilterToScan( ColumnFilter cf, HBaseValueMeta columnMeta, VariableSpace vars, boolean matchAny )
+    throws Exception {
+    delegate.addColumnFilterToScan( cf, columnMeta, vars, matchAny );
+  }
+
+  /*
+   * (non-Javadoc)
+   *
+   * @see org.pentaho.hadoop.shim.mapr31.delegatingShims.HBaseConnectionInterface#addColumnToScan(java.lang.String,
```

Comment:
Should not refer to mapr31 shim in Javadoc (many instances of this throughout the file)